### PR TITLE
updated AD apis

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/ad/api.md [hunk 1]
```diff
@@ -162,24 +173,69 @@ POST _opendistro/_anomaly_detection/detectors/<detector_id>/_preview
...
+          "feature_name": "total_order",
+          "data": 489.9929131106
         }
       ],
-      "data_start_time": 1573514220000,
-      "data_end_time": 1573514520000
+      "anomaly_grade": 0,
+      "confidence": 0.99
     }
   ],
```
Comment (ylwu-amzn) [inline docs/ad/api.md:189]: anomaly_result has multiple items, should we add `...` before `]` ?

File: docs/ad/api.md [hunk 2]
```diff
@@ -649,101 +830,278 @@ GET _opendistro/_anomaly_detection/stats/<stat>
...
-      "models": [
+    "cDcGNsPoRAyRMlPP1m-vZw" : {
+      "ad_execute_request_count" : 0,
+      "ad_execute_failure_count" : 0,
+      "models" : [
         {
-          "detector_id": "asbPyG4BlFrvFHzloDO-",
-          "model_type": "rcf",
-          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_2"
+          "detector_id" : "asbPyG4BlFrvFHzloDO-",
```
Comment (ylwu-amzn) [inline docs/ad/api.md:870]: How about we replace all `asbPyG4BlFrvFHzloDO-` with `m4ccEnIBTXsGi3mvMt9p` in this example? As we are using this detector in other sample APIs. I have changed in public API example doc. You can copy directly.

File: docs/ad/api.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/ad/api.md:191]: Added.

File: docs/ad/api.md (hunk 2 above)
Comment (ashwinkumar12345) [reply docs/ad/api.md]: Done.

**Pull request merged**